react.gradle

```diff
@@ -315,6 +315,7 @@ afterEvaluate {
         // mergeResources task runs before the bundle file is copied to the intermediate asset directory from Android plugin 4.1+.
         // This ensures to copy the bundle file before mergeResources task starts
         def mergeResourcesTask = tasks.findByName("merge${targetName}Resources")
+        currentAssetsCopyTask.dependsOn(currentBundleTask)
```

Comment:
Isn't this line equivalent to this other?

https://github.com/facebook/react-native/blob/2e8893fb5be0c877e0818d04264d75fa92ae0f8e/react.gradle#L312

Reply:
Yeah, I think that's equivalent.

I have tried to replicate the issue at RN 65.1 then the issue doesn't appear.
I will close this PR.

Thanks 🍻